# This message is for storing bounding boxes in a generic format
float64 x1
float64 y1
float64 x2
float64 y2
float64 score
int16 class_id